src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -209,38 +210,60 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
+                        <div className="fw-4 fs-11 lh-16 flex">
+                            <World className="icon-dim-16 mr-4 scg-5" />
+                            Active on <span className="fw-6 ml-4">{this.props.envName} </span>
+                        </div>
                     </span>
+                )}
+                {mat.runningOnParentCd && (
+                    <span className="bcg-1 br-4 eg-2 cn-9 pt-3 pb-3 pl-6 pr-6 bw-1 mr-6">
+                        <div className="fw-4 fs-11 lh-16 flex">
+                            <World className="icon-dim-16 mr-4 scg-5" />
+                            Active on <span className="fw-6 ml-4">{this.props.parentEnvironmentName}</span>
+                        </div>
+                    </span>
+                )}
+            </>
+        )
+    }
+
+    renderProgressingCD = (mat) => {
+        return (
+            <span className="bcy-1 br-4 ey-2 cn-9 pt-3 pb-3 pl-6 pr-6 bw-1 mr-6">
+                <div className="fw-4 fs-11 lh-16 flex">
+                    <div className={`dc__app-summary__icon icon-dim-16 mr-6 progressing progressing--node`}></div>
+                    Deploying on <span className="fw-6 ml-4">{this.props.envName} </span>
                 </div>
-            )
-        } else if (mat.latest) {
-            return (
-                <div className="bcv-1 pt-6 pb-6 pl-16 pr-16 br-4">
-                    <span className="cn-9 fw-6">Deployed on </span>
-                    <span className="cv-5 fw-6">{this.props.envName} </span>
+            </span>
+        )
+    }
+
+    renderFailedCD = (mat) => {
+        return (
+            <span className="bcr-1 br-4 er-2 cn-9 pt-3 pb-3 pl-6 pr-6 bw-1 mr-6">
+                <div className="fw-4 fs-11 lh-16 flex">
+                    <Failed className="icon-dim-16 mr-4" />
+                    Last deployment failed on <span className="fw-6 ml-4">{this.props.envName} </span>
                 </div>
-            )
-        } else if (mat.runningOnParentCd) {
+            </span>
+        )
+    }
+
+    renderSequentialCDCardTitle = (mat) => {
+        if (this.props.stageType !== STAGE_TYPE.CD) return
+
+        if (mat.latest || mat.runningOnParentCd || mat.artifactStatus === 'Progressing' || mat.artifactStatus === 'Degraded' || mat.artifactStatus === 'Failed') {
             return (
-                <div className="bcv-1 pt-6 pb-6 pl-16 pr-16 br-4">
-                    <span className="cn-9 fw-6">Deployed on </span>
-                    <span className="cv-5 fw-6">{this.props.parentEnvironmentName}</span>
+                <div className="bcn-0 p-8 br-4 dc__border-bottom flex left">
+                    {this.renderActiveCD(mat)}
+                    {mat.artifactStatus === 'Progressing' && this.renderProgressingCD(mat)}
+                    {mat.artifactStatus === 'Degraded' || (mat.artifactStatus === 'Failed' && this.renderFailedCD(mat))}
```

Comment:
Fix the condition wrapper. It should be `(mat.artifactStatus === 'Degraded' || mat.artifactStatus === 'Failed') && this.renderFailedCD(mat)`.